lib/logstash/codecs/cef.rb

```diff
@@ -114,6 +117,9 @@ def decode(data)
     end
 
     yield event
+  rescue => e
+    @logger.error("Failed to decode event payload. Generating failure event with payload in message field.", :error => e.message, :backtrace => e.backtrace, :data => data)
+    yield LogStash::Event.new("message" => data, "tags" => ["_cefparsefailure"])
```

Comment:
they most used one (codec json) doesn't allow it, so I'm not that concerned, maybe we can have this as is and eventually add it if needed?